### PR TITLE
task 10

### PR DESCRIPTION
**Pull request by Adouleur**
Implement Clear Completed Button

### CODE REVIEW

File: src/constants/constants.js [hunk 1]
```diff
@@ -0,0 +1 @@
+export const filters = Object.freeze(['all', 'active', 'completed']);
```
Comment (JisonDinAlt) [inline src/constants/constants.js]: Does you need Object.freeze there?

**Review by JisonDinAlt — CHANGES_REQUESTED** [review]

File: src/components/molecules/InfoBoard.vue [hunk 2]
```diff
@@ -1,23 +1,33 @@
...
+        :key="filter"
+        :class="{ active: currentFilter === filter }"
+        @click="setFilter(filter)"
+      >
+        {{ capitalizeFirstLetter(filter) }}
+      </ButtonComponent>
+
+      <ButtonComponent @click="clearCompleted">Clear Completed</ButtonComponent>
     </div>
     <div class="tasks-left">{{ tasksLeft }} {{ tasksLeft === 1 ? 'task' : 'tasks' }} left</div>
```
Comment (KHarpushkina) [inline src/components/molecules/InfoBoard.vue]: Could you move this calculation to the computed?

**Review by KHarpushkina — CHANGES_REQUESTED** [review]

File: src/constants/constants.js (hunk 1 above)
Comment (Adouleur) [reply src/constants/constants.js]: Removed, thank you!

File: src/components/molecules/InfoBoard.vue (hunk 2 above)
Comment (Adouleur) [reply src/components/molecules/InfoBoard.vue]: Refactored, thank you!

**Pull request merged**